diff --git a/crates/editor/src/authoring_windows.rs b/crates/editor/src/authoring_windows.rs
index 311fdbbc73119105592bf86a7c4757beb1f49f6a..dc97b61bfea61c53c79c7b4d85ab56c625858acf 100644
--- a/crates/editor/src/authoring_windows.rs
+++ b/crates/editor/src/authoring_windows.rs
@@ -122,8 +122,8 @@ impl AuthoringWindows {
             #[cfg(feature = "visual-validation")]
             let window = window
                 .default_width(610.0)
-                .default_height(620.0)
-                .default_pos(egui::pos2(650.0, 42.0));
+                .default_height(700.0)
+                .default_pos(egui::pos2(650.0, 10.0));
             window.show(context, |ui| {
                 egui::ScrollArea::both().show(ui, |ui| {
                     sequencer.show(ui, &permissions);
